package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "node-firefox-forward-ports",
+  "version": "1.0.0",
+  "description": "Find attached FirefoxOS devices using ADB",
+  "main": "index.js",
+  "dependencies": {
+    "adbkit": "^2.1.6",
+    "es6-promise": "^2.0.1",
+    "portfinder": "^0.4.0"
+  },
+  "devDependencies": {
+    "gulp": "^3.8.10",
+    "gulp-jscs": "^1.3.1",
```

Comment:
Yup, this is copypasta from another node-firefox module. Most of them have all this stuff in `devDependencies`.

Removing all except `gulp` and reinstalling seems to work fine. Might be worth circling back to the other modules and doing the same